From 050233b0998b381c188d09e5521660ae45ac02cc Mon Sep 17 00:00:00 2001
From: Joshua Strobl <joshua@stroblindustries.com>
Date: Sun, 8 Oct 2017 22:25:25 +0300
Subject: [PATCH 1/1] gdm: Allow empty password text

Not particularly useful, livecds are passwordless and thus not allowing empty password is unhelpful.
---
 js/gdm/authPrompt.js | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/js/gdm/authPrompt.js b/js/gdm/authPrompt.js
index a436431..b1c518f 100644
--- a/js/gdm/authPrompt.js
+++ b/js/gdm/authPrompt.js
@@ -181,14 +181,14 @@ var AuthPrompt = new Lang.Class({
                               x_align: St.Align.END,
                               y_align: St.Align.END });
 
-        this._updateNextButtonSensitivity(this._entry.text.length > 0);
+        this._updateNextButtonSensitivity(true);
 
         this._entry.clutter_text.connect('text-changed',
                                          Lang.bind(this, function() {
                                              if (!this._userVerifier.hasPendingMessages)
                                                  this._fadeOutMessage();
 
-                                             this._updateNextButtonSensitivity(this._entry.text.length > 0 || this.verificationStatus == AuthPromptStatus.VERIFYING);
+                                             this._updateNextButtonSensitivity(true);
                                          }));
         this._entry.clutter_text.connect('activate', Lang.bind(this, function() {
             if (this.nextButton.reactive)
@@ -416,7 +416,7 @@ var AuthPrompt = new Lang.Class({
     },
 
     updateSensitivity: function(sensitive) {
-        this._updateNextButtonSensitivity(sensitive && (this._entry.text.length > 0 || this.verificationStatus == AuthPromptStatus.VERIFYING));
+        this._updateNextButtonSensitivity(sensitive);
         this._entry.reactive = sensitive;
         this._entry.clutter_text.editable = sensitive;
     },
-- 
2.14.2

